test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -39,13 +34,15 @@ const (
 	threshold60K = 30
 )
 
+
 // TestSchedule100Node3KPods schedules 3k pods on 100 nodes.
```

Comment:
are we still relying on defaults here? we need the 100, 3000 to be explicit *please* :) . we can add defaults later...